h/api/storage.py

```diff
@@ -37,10 +36,13 @@ def fetch_annotation(id):
     :returns: the annotation, if found, or None.
     :rtype: dict, NoneType
     """
-    return models.Annotation.fetch(id)
+    if _postgres_enabled(request):
```

Comment:
Hmm, although I see you're using `_postgres_enabled` as a test seam. Happy to leave this as-is for now.